api/HTMLIFrameElement.json

```diff
@@ -375,10 +375,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
```

Comment:
OK, so https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/csp / https://w3c.github.io/webappsec-cspee/#dom-htmliframeelement-csp seems to not be in Firefox or Safari. https://wpt.fyi/results/content-security-policy/embedded-enforcement/iframe-csp-attribute.html agrees.

Oh well :(